DEV_DOCS.md

```diff
@@ -169,4 +169,15 @@ If a translation of a particular string is not yet available, then auspice will
 
 ## Releases & versioning
 
-New versions are released via the `./releaseNewVersion.sh` script from an up-to-date `master` branch. It will prompt you for the version number increase, push changes to the `release` branch and, as long as the GitHub workflow is successful then a new version will be automatically published to [npm](https://www.npmjs.com/package/auspice).
+1. Run the `./releaseNewVersion.sh` script from an up-to-date `master` branch. It will prompt you for the version number increase, push changes to the `release` branch and, as long as the GitHub workflow is successful then a new version will be automatically published to [npm](https://www.npmjs.com/package/auspice).
+2. Follow instructions in the [nextstrain/bioconda-recipes README](https://github.com/nextstrain/bioconda-recipes#readme) to update the version on Bioconda. Make the following changes in `recipes/auspice/meta.yaml`:
```

Comment:
What's in the README? I think this is over-complicated.

One just needs to go to the bioconda recipe directly in the bioconda repo - click on edit and do it right there - then make a PR from there. Minimal number of clicks - no?

Reply:
Here's a direct link to click, takes you straight to the editor, just need to increase the number and copy the hash:
https://github.com/bioconda/bioconda-recipes/edit/master/recipes/auspice/meta.yaml

Reply:
I'd be in favor of this to reduce tedium. Changed in 5c5555599038ea76eec1bc7b8f10b4b8d238f3b2, but I'd like to hear thoughts from @tsibley or others that have historically used [nextstrain/bioconda-recipes](https://github.com/nextstrain/bioconda-recipes) to create Bioconda PRs.

Reply:
Sure!

Reply:
One benefit of using [nextstrain/bioconda-recipes](https://github.com/nextstrain/bioconda-recipes) to create PRs is that anyone on the team can add/edit commits in the PR. This is still possible with PRs from personal forks, but you'd have to explicitly grant access to others on your fork which is a bit extra I think.

Reply:
For longer lasting forks, like that of bioconda-recipes, I generally prefer to do it as the org rather than my personal user. This helps me compartmentalize personal vs. work stuff, lends the imprimatur of Nextstrain to the PRs, and lets everyone on the team contribute to the PRs (e.g. if I go on vacation).

That said, I don't personally mind instructions that assume personal forks as I can still choose to use the org fork if I continue to want to.

Reply:
Added this for a good balance: 1bd4175914da8f823898a1af02da01ba36a4fea1